### PR TITLE
Added new section for contributors

### PR DESCRIPTION
**Pull request by abbyad**
Brings in workflow and style guides for code and docs into the new **Contribute** section. Also updates the docs style guide to be consistent with itself.

This covers work previously done by @CescaGarrett, @MaxDiz, and @garethbowen, so would be great to get a review from you all.

Note the new structure:
image

### CODE REVIEW

File: content/en/contribute/_index.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+The [Development](https://forum.communityhealthtoolkit.org/c/developement/7) section of the forum is a great place to introduce yourself and ask questions. Or you can also jump right in:
+- Issues labeled [help wanted](https://github.com/medic/cht-core/issues?q=is%3Aopen%20is%3Aissue%20label%3A%22Help%20wanted%22) are a great place to start. 
+- Learn to [build and host apps]({{% ref "apps" %}}), and share your experiences
+- [Share a new idea or question](https://forum.communityhealthtoolkit.org/) on our Forum
+
+**Are you a researcher, designer, or health practitioner?**
+
+- [Share your expertise](https://forum.communityhealthtoolkit.org) on our Forum
+- Sign up for our [newsletter](https://communityhealthtoolkit.org/contact) to get the latest updates
```
Comment (garethbowen) [inline content/en/contribute/_index.md:21]: And follow us on Twitter, facebook, etc?

File: content/en/contribute/_index.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+title: "Contributing to the Community Health Toolkit"
+linkTitle: "Contributing"
+weight: 10
+description: >
+  How to contribute to the CHT tools and documentation
+---
+
+There are many ways to contribute to the Community Health Toolkit.
+
+**Are You A Developer?**
```
Comment (garethbowen) [inline content/en/contribute/_index.md:11]: Title case is out of place here.
```suggestion
**Are you a developer?**
```

File: content/en/contribute/code/workflow.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ description: >
 
 ### Writing
 
-Where possible, follow our coding [style guide]({{< ref "docs-style-guide" >}}).
+Where possible, follow our coding [style guide]({{< ref "contribute/docs/style-guide" >}}).
```
Comment (garethbowen) [inline content/en/contribute/code/workflow.md:13]: As @mrjones-plip noticed in #165 , this should link to the coding style guide not the docs style guide.

```suggestion
Where possible, follow our [coding style guide]({{< ref "contribute/code/style-guide" >}}).
```

File: content/en/contribute/_index.md (hunk 2 above)
Comment (abbyad) [reply content/en/contribute/_index.md:11]: Good call!

File: content/en/contribute/_index.md (hunk 1 above)
Comment (abbyad) [reply content/en/contribute/_index.md:21]: I was going to add them based on your suggestion, then realized we don't mention them on the main CHT site. We can add them later if they become active.

**Comment by MaxDiz** [comment]
Not sure if this is related, but for some reason I am getting a phantom "workflow" showing on my local index
Screen Shot 2020-06-22 at 5 50 19 PM

**Comment by abbyad** [comment]
Thanks @MaxDiz. Does it disappear if you stop and restart your local Hugo? One way to avoid the phantom (zombie?) menu links is to run Hugo with the flag to disable fast rendering: eg `hugo server --disableFastRender`

**Pull request merged**